src/main/java/uk/gov/hmcts/ecm/common/client/CcdClient.java

```diff
@@ -90,9 +90,10 @@ public CCDRequest startCaseCreation(String authToken, CaseDetails caseDetails) t
         return restTemplate.exchange(uri, HttpMethod.GET, request, CCDRequest.class).getBody();
     }
 
-    public CCDRequest startCaseCreationTransfer(String authToken, CaseDetails caseDetails) throws IOException {
-        HttpEntity<String> request =
-                new HttpEntity<>(buildHeaders(authToken));
+    public CCDRequest startCaseCreationTransfer(String authToken,
+                                                uk.gov.hmcts.ecm.common.model.ccd.CaseDetails caseDetails)
+        throws IOException {
```

Comment:
I think this is the change for all case transfers and not only for Reform to ECM? Was there any problem in existing case transfers in ECM too?